### PR TITLE
[NG] New Image Hub Page

### PR DESCRIPTION
**Pull request by Jashpatel1**
**Description**

This PR updates Image Hub Page design.
https://deploy-preview-1348--layer5ng.netlify.app/projects/image-hub

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Image-Hub/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                                </a>
+                            </div>
+                        </Slider>
+                        <p>Consul Service Mesh Architecture w/WebAssembly</p>
+                    </div>
+                    <div className="contributors">
+                        <h1>Contributors</h1>
+                        <p>This demo application has been enabled by experimental works of Nic Jackson of HashiCorp, and Kanishkar J, Lee Calcote, and other contributors of Layer5.
+        Learn more about the Layer5 and HashiCorp Partnership.</p>
+                        <Button primary title="Learn More"></Button>
```
Comment (Jashpatel1) [inline src/sections/Image-Hub/index.js:65]: @leecalcote What should this button point to?

Comment (leecalcote) [reply src/sections/Image-Hub/index.js:65]: Yes, the blog post on the subject. Please.

Comment (Jashpatel1) [reply src/sections/Image-Hub/index.js:65]: I couldn't find the blog post. Redirecting to partners page for now.

Comment (Nikhil-Ladha) [reply src/sections/Image-Hub/index.js:65]: https://deploy-preview-1348--layer5ng.netlify.app/blog/programs/starting-smi-conformance-testing-with-meshery

Comment (Jashpatel1) [reply src/sections/Image-Hub/index.js]: I think this is the wrong link.

Comment (Nikhil-Ladha) [reply src/sections/Image-Hub/index.js]: Ahhh....I was thinking this is the SMI page PR😆

**Comment by Jashpatel1** [comment]
@leecalcote @Nikhil-Ladha @Tanuj22 This PR is ready to get merged. Please have a look.

**Comment by Nikhil-Ladha** [comment]
There are few mobile responsiveness issues, please fix them.
- The video is getting cut off
- The slider arrows needs some margin, from the slides.

**Comment by Jashpatel1** [comment]
> There are few mobile responsiveness issues, please fix them.
> 
> * The video is getting cut off

Only the thumbnail is getting cut off, not the video.

**Comment by Tanuj22** [comment]
> > There are few mobile responsiveness issues, please fix them.
> > 
> > * The video is getting cut off
> 
> Only the thumbnail is getting cut off, not the video.

It's getting cut off from both sides

**Comment by Tanuj22** [comment]
Oh no it's just the thumbnail. But even that shouldn't be cut off 🤔

**Comment by Jashpatel1** [comment]
> Oh no it's just the thumbnail. But even that shouldn't be cut off

Done

**Comment by Tanuj22** [comment]
> > Oh no it's just the thumbnail. But even that shouldn't be cut off
> 
> Done

Still sticking from around 900px

**Review by Tanuj22 — APPROVED** [review]
LGTM

**Pull request merged**